File: Java/Week_3-4/src/models/Subject.java
package models;

public class Subject
{
    public String subjectId;
    public String subjectName;
    public String deptId;
    public int    credits;

    public Subject(String subjectId, String subjectName, String deptId, int credits)
    {
        this.subjectId   = subjectId;
        this.subjectName = subjectName;
        this.deptId      = deptId;
        this.credits     = credits;
    }

    @Override public String toString()
    {return subjectName + " [" + subjectId + "]";}
}

File: PythonProject/Week 2/models/subject.py
class Subject:
    def __init__(self, subject_id, subject_name, dept_id, credits, semester_offered, max_marks):
        self.subject_id = subject_id
        self.subject_name = subject_name
        self.dept_id = int(dept_id)
        self.credits = int(credits)
        self.semester_offered = int(semester_offered)
        self.max_marks = int(max_marks)

    def __repr__(self):
        return f"Subject({self.subject_id}, {self.subject_name})"
